src/main/java/org/broadinstitute/hellbender/utils/test/IntegrationTestSpec.java

```diff
@@ -28,11 +30,17 @@
     private final int nOutputFiles;
     private final List<String> expectedFileNames;
 
+    //If this field is set to true, bam files will be compared after they get sorted.
+    //This is needed as a workaround because Spark tools don't respect a pre-ordered BAMs
+    // and so may create BAMs that are sorted differently than the input (though both orders will be valid).
```

Comment:
Are we ever producing bams in spark that claim to be coordinate-sorted but aren't?